deps.windows/90-vlc.ps1

```diff
@@ -0,0 +1,54 @@
+param(
+    [string] $Name = 'vlc',
+    [string] $Version = '3.0.8',
+    [string] $Uri = 'https://cdn-fastly.obsproject.com/downloads/vlc.zip',
+    [string] $Hash = "${PSScriptRoot}/checksums/vlc.zip.sha256"
```

Comment:
We actually use the git repo on Windows, because we wanted the latest headers. If nothing else, we should be using latest stable 3.x.

Reply:
You mean the contents of the zip are actually just a copy of the git repo?

Reply:
Oh. I have no idea what vlc.zip on fastly actually contains, now that you mention it. For some reason, I thought we just did a git checkout on CI. Inspecting it, it does appear to be some zip of the VLC source, and I'm not sure which version. We just mark it as 3.0.0-git on CI.

Reply:
We _could_ just use a git checkout of 3.0.8 then?

Reply:
For now, let's just do what this PR does since that's what obs-studio CI does. We can change it later to either do a git checkout or a source download of some kind. If that becomes unreliable in the future for some reason (server outages), we can make a mirror again then.